lib/utils/getlogger.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/**
+ * @returns {Logger}
+ */
+module.exports = function getLogger() {
+	const logsCollection = new Map( [
+		[ 'info', [] ],
+		[ 'error', [] ]
+	] );
+
+	let lastInserted;
+
+	const logger = {
+		/**
+		 * @param {String} msg
+		 */
+		info( msg ) {
+			return logger._log( 'info', msg );
+		},
+
+		/**
+		 * @param {String|Error} msg
+		 */
+		error( msg ) {
+			if ( msg instanceof Error ) {
+				msg = msg.stack;
+			}
+
+			return logger._log( 'error', msg );
+		},
+
+		/**
+		 * @param {Logs} responseLogs
```

Comment:
Wrong variable name or method name or both. Why would logger have to know that what you pass it is response logs and not any logs? And `concat()` I've got some doubts whether `concat()` is the right name too, but I can't propose something better.